mmbot/mmbot-tx.py

```diff
@@ -197,16 +201,17 @@ def parse_transfer(txn: Transaction, user: str):
     return txn.narration
 
 
-def main_transaction_handling(added_transactions: Transaction, mm_url, token):
+def process_transaction(beancount_transaction: Transaction, mm_url, token):
     # Get the correct member name from the transaction
-    member_names = get_users_from_tx(added_transactions)
+    member_names = get_users_from_tx(beancount_transaction)
     logging.debug('Membernames from transaction: %s', member_names)
     mm_users = get_mm_users(member_names)
 
     if not mm_users:
+        logging.info("Did not find mattermost user for the transaction")
```

Comment:
I would make this logging.**info** so we can see at runtime in the output of the github action if a typo was made in the `mm_name` in the `members.beancount` file.